website/thaliawebsite/settings/settings.py

```diff
@@ -156,8 +156,11 @@
 CORS_URLS_REGEX = r"^/(?:api|user/oauth)/.*"
 
 # OAuth configuration
+OAUTH_SCHEME = os.getenv("OAUTH_SCHEME", "nu.thalia")
```

Comment:
Maybe rename this to APP_OAUTH_SCHEME to signify it's intended usage

Reply:
done